client/multi-currency-analytics/index.js

```diff
@@ -122,3 +122,44 @@ addFilter(
 		return tableData;
 	}
 );
+
+addFilter(
+	'woocommerce_admin_orders_report_filters',
+	'woocommerce-payments',
+	( filters ) => [
+		{
+			label: __( 'Currency', 'woocommerce-payments' ),
```

Comment:
What do you think about labelling this `Customer currency` just to make it clear that it refers to the currency the order was placed in?

Reply:
👍 I used `Currency` following the design, but it can be confusing as we're actually filtering and converting the order currency. Addressed in 881d882

Reply:
Thanks for addressing!